src/template-linter.ts

```diff
@@ -221,17 +221,27 @@ export default class TemplateLinter {
   }
   async getFindUp(): Promise<FindUp> {
     if (!this._findUp) {
-      const { findUp } = await eval(`import('find-up')`);
+      const { findUp } = await import('find-up');
```

Comment:
we use eval here to not bundle this part using webpack, our target remains cjs

Reply:
why would webpack change how this works at all? :sweat_smile: 

I'll revert -- it belongs in another PR anyway